test/tests/imageserver/test_image_service_system.py

```diff
@@ -0,0 +1,319 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+This test validate the system level function of RackHD image-service. This test include VMware ESXi install and node
+rediscover.It use image server to store ESXi image and microkernel used for RackHD discovery.
+You need put an config file in the /config directory
+'''
+
+import os
+import sys
+import time
+import flogging
+import random
+import json
+import fit_common
+import pexpect
+import unittest
+import test_api_utils
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+
+try:
+    FILE_CONFIG = json.loads(open(fit_common.CONFIG_PATH + "fileserver_config.json").read())
+except BaseException:

```

Comment:
Using the updates in PR 4629, the file can be passed in via the run_tests.py command line, and accessed via fitcfg()['myfile'].